src/libcalamaresui/utils/Paste.cpp

```diff
@@ -30,7 +30,7 @@ using namespace CalamaresUtils::Units;
  * Returns an empty QByteArray() on any kind of error.
  */
 STATICTEST QByteArray
-logFileContents()
+logFileContents( qint64 sizeLimit )
```

Comment:
Since this is intended to be in KiB, it's probably useful to name it `sizeLimitKiB` or so.